app/src/components/common/navbar/NavBar.js

```diff
@@ -1,32 +1,17 @@
-import Link from "next/link";
+import DesktopNavBar from "@/components/common/navbar/DesktopNavBar";
+import MobileNavBar from "@/components/common/navbar/MobileNavBar";
 
 export default function NavBar() {
   return (
-    <div className="bg-[#b7997b] flex p-10 justify-between poppins">
-      <div className="">
-        <Link className="text-xl text-black no-underline pr-10" href="/about">
-          SOBRE MIM
-        </Link>
-        <Link
-          className="text-xl text-black no-underline pr-10"
-          href="/contacts"
-        >
-          CONTACTOS
-        </Link>
-      </div>
-      <div className="">
-        <Link
-          className="text-xl text-black no-underline pr-10"
-          href="/services"
-        >
-          SERVIÇOS
-        </Link>
-        <Link
-          className="text-xl text-black no-underline pr-10"
-          href="/projects"
-        >
-          PROJETOS
-        </Link>
+    <div>
+      <div>
+        <div className="menu-desktop">
```

Comment:
im not sure that actually having both navbars here is the best practice

where did you get the inspiration for this?

this seems a better approach: https://blog.logrocket.com/create-responsive-navbar-react-css/

you can also use some sort of variable to conditionally render the navbar you need

Reply:
here: https://khuang159.medium.com/creating-a-hamburger-menu-in-react-f22e5ae442cb

The approach referred in your suggestion demands the installation of a new package react-responsive and I decided to proceed with a simpler option (no package needed). However, I am going to investigate more options